make/mshadow.mk

```diff
@@ -40,15 +42,17 @@ ifndef USE_F16C
         ifeq ($(detected_OS),Linux)
             F16C_SUPP = $(shell cat /proc/cpuinfo | grep flags | grep f16c)
         endif
-	ifneq ($(F16C_SUPP), NONE)
+	ifneq ($(F16C_SUPP),)
```

Comment:
Should we be using strip so that whitespace doesn't evaluate to non empty: https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html

Reply:
If we're requiring this flag be explicitly disabled for cross-compilation this will be a breaking change for a few users of MXNet.

Reply:
Changed